msi/GitProduct.wxs

```diff
@@ -264,6 +264,8 @@
   </Fragment>
 
   <Fragment>
+    <SetProperty Id="ExecConfigureSslCAInfo" Value='"[#BashExe]" -c "git config -f \"/[#GfwGitConfig]\" http.sslCAinfo \"/[#ca_bundle.crt]\""' Sequence="execute" Before="ExecConfigureSslCAInfo" />
```

Comment:
If I understand correctly, this change will repeat InnoSetup-based installer's mistake of putting a Git for Windows-specific path into ProgramData? Please don't. I intend to fix this in the InnoSetup-based installer, still. The ProgramData config should be valid, still, even after Git for Windows was uninstalled.